### DISCUSSION TITLE
Need help for build the mono repo and configure dev environement

### DISCUSSION THREAD

**Discussion by HumansIt** [medusajs/medusa#4980] [Q&A]
We have a lot of errors when we clone the repo and follow the procedure as described in the documentation. 
Is there somebody who can help us.

error on yarn install :
![image](https://github.com/medusajs/medusa/assets/117761885/d0e2b108-98aa-4a2a-be83-f37e6a7a0492)

**Comment by pevey** [comment]
You need to run `yarn install` to download all the packages before you run `yarn build`. 

Also, if you are on Windows, you will need to install WSL, along with the Linux flavor of your choice.  Debian or Ubuntu are really good options.  Ubuntu is the default, so if you are okay with that, just install WSL and accept all the defaults. https://learn.microsoft.com/en-us/windows/wsl/install

Medusa is not developed or tested on Windows, so if you try to develop on Windows native, some things may break unexpectedly.  It will save you time and hassle to set up WSL and install the WSL plugins for vscode to allow you to treat it just like an extension of your Windows machine.  It is integrated very well at this point.
